relationships/synthesis/EXT-SERVICE-to-INFRA-ELASTICACHEMEMCACHEDCLUSTER.yml

```diff
@@ -0,0 +1,56 @@
+relationships:
+  - name: extServiceCallsInfraElasticacheMemcachedCluster
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Metric" ]
+      - attribute: db.system
+        anyOf: [ "memcached" ]
+    relationship:
+      expires: P75M
+      relationshipType: CALLS
+      source:
+        extractGuid:
+          attribute: entity.guid
+      target:
+        lookupGuid:
+          candidateCategory: AWSELASTICACHEMEMCACHEDCLUSTER
+          fields:
+            - field: cloud.resource_endpoint_address
+              attribute: net.peer.name
+            - field: cloud.resource_endpoint_port
+              attribute: net.peer.port
+            - field: cloud.elasticache_engine
+              attribute: db.system
+
+
+  - name: apmServiceCallsInfraElastiCacheMemcachedCluster
+    version: "1"
+    origins:
+      - APM Metrics
+    conditions:
```

Comment:
Looks like we have data matching this condition: https://staging.onenr.io/02wdv5BXEjE
But the regex looks incorrect: https://staging.onenr.io/0bRK94GrzQE

Reply:
I think that the problem is the `9` after the final `/`: https://staging.onenr.io/0EjOe7YaDw6

Reply:
Please, in your next PR, include these kind of queries in the description. It simplifies a lot the review process.